pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -656,16 +689,21 @@ func (m *kubeGenericRuntimeManager) computePodResizeAction(ctx context.Context,
 		return true
 	}
 
-	actuatedResources, found := m.actuatedState.GetContainerResources(pod.UID, container.Name)
+	actuatedContainerResources, found := m.actuatedState.GetContainerResources(pod.UID, container.Name)
 	if !found {
 		logger.Error(nil, "Missing actuated resource record", "pod", klog.KObj(pod), "container", container.Name)
 		// Proceed with the zero-value actuated resources. For restart NotRequired, this may
 		// result in an extra call to UpdateContainerResources, but that call should be idempotent.
 		// For RestartContainer, this may trigger a container restart.
 	}
 
-	desiredResources := containerResourcesFromRequirements(&container.Resources)
-	currentResources := containerResourcesFromRequirements(&actuatedResources)
+	var actuatedPodResources *v1.ResourceRequirements
+	if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodLevelResourcesVerticalScaling) {
+		actuatedPodResources, _ = m.actuatedState.GetPodLevelResources(pod.UID)
```

Comment:
do we need to check the return value of `found` here (similar to what we do with container resources above)

Reply:
It's just used for logging, but probably good to know?